components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -477,6 +479,14 @@ protected void processAuthenticationResponse(HttpServletRequest request, HttpSer
                             errorMessage);
                 }
 
+                String idpName = context.getExternalIdP().getIdPName();
```

Comment:
In a federated scenario, is there any change for context.getExternalIdP() to return a null?